src/components/page/StandardPage.tsx

```diff
@@ -396,49 +393,54 @@ const StandardPageInner = <T,>({
           <ToolbarContent>
-            <ToolbarToggleGroup toggleIcon={<FilterIcon />} breakpoint="xl">
-              {primaryFilters.length > 0 && (
-                <FilterGroup
-                  fieldFilters={primaryFilters}
-                  onFilterUpdate={setSelectedFilters}
-                  selectedFilters={selectedFilters}
-                  supportedFilterTypes={supportedFilters}
+            <Split hasGutter>
+              {selectedIds && onSelect && (
+                <TableBulkSelect
+                  selectedIds={selectedIds}
+                  dataIds={filteredData?.map(toId)}
+                  pageDataIds={pageData?.map(toId)}
+                  onSelect={onSelect}
                 />
               )}
-              <AttributeValueFilter
-                fieldFilters={fields
-                  .filter(({ filter }) => filter && !filter.primary && !filter.standalone)
-                  .map(toFieldFilter(flatData))}
-                onFilterUpdate={setSelectedFilters}
-                selectedFilters={selectedFilters}
-                supportedFilterTypes={supportedFilters}
-              />
-              {Boolean(fields.find((field) => field.filter?.standalone)) && (
-                <FilterGroup
+
+              <ToolbarToggleGroup toggleIcon={<FilterIcon />} breakpoint="xl">
+                {primaryFilters.length > 0 && (
+                  <FilterGroup
+                    fieldFilters={primaryFilters}
+                    onFilterUpdate={setSelectedFilters}
+                    selectedFilters={selectedFilters}
+                    supportedFilterTypes={supportedFilters}
+                  />
+                )}
+                <AttributeValueFilter
                   fieldFilters={fields
-                    .filter((field) => field.filter?.standalone)
+                    .filter(({ filter }) => filter && !filter.primary && !filter.standalone)
                     .map(toFieldFilter(flatData))}
                   onFilterUpdate={setSelectedFilters}
                   selectedFilters={selectedFilters}
                   supportedFilterTypes={supportedFilters}
                 />
-              )}
-              <ManageColumnsToolbar
-                resourceFields={fields}
-                defaultColumns={fieldsMetadata}
-                setColumns={setFields}
-              />
-              {GlobalActionToolbarItems?.length > 0 &&
-                GlobalActionToolbarItems.map((Action, index) => (
-                  <Action key={index} dataOnScreen={showPagination ? pageData : filteredData} />
-                ))}
-            </ToolbarToggleGroup>
-
-            {selectedCountLabel && (
-              <ToolbarItem className="forklift-page__toolbar-item__selected-count">
-                {selectedCountLabel(selectedIds.length ?? 0)}
-              </ToolbarItem>
-            )}
+                {!!fields.find((field) => field.filter?.standalone) && (
```

Comment:
```suggestion
                {Boolean(fields.find((field) => field.filter?.standalone)) && (
```

Reply:
Looks like this was missed in the rebase, will add this back.